src/perf_tool_backend.ml

```diff
@@ -33,8 +34,15 @@ module Record_opts = struct
           "Record a program's full execution instead of using a snapshot ring buffer.\n\
            Warning: The trace grows at a rate of hundreds of megabytes per second. \
            Traces larger than 100 MiB are likely to crash the trace viewer."
+    and snapshot_size =
+      Pow2_bytes.optional_flag
+        "-snapshot-size"
+        ~doc:
+          " Tunes the amount of data captured in a trace. By default, 4M for root or if \
```

Comment:
Nit: we use mebibyte units above, let's be consistent one way or another (weak preference for mebibytes)

Reply:
Nit: "cause the trace viewer to hit your browser's per-tab memory limit and fail to load the trace" or something, it doesn't *really* crash.

Reply:
I don't know if `Bytes_unit` can take -bibyte, though...

Reply:
`Byte_units` can't, so I'm going to use its conventions universally (10M, 50G, etc.)

Reply:
> Nit: "cause the trace viewer to hit your browser's per-tab memory limit and fail to load the trace" or something, it doesn't _really_ crash.

I mean... as far the user is concerned, it does. I don't feel like being more specific is particularly helpful here, and the help text is already to long imo.

Reply:
shannon suggested "fail to load" instead of "crash"